airbyte-cdk/python/airbyte_cdk/sources/streams/http/rate_limiting.py

```diff
@@ -21,6 +21,8 @@
 def default_backoff_handler(max_tries: int, factor: int, **kwargs):
     def log_retry_attempt(details):
         _, exc, _ = sys.exc_info()
+        if exc.response is not None:
+            logger.info(f"Status code: {exc.response.status_code}, Response Content: {exc.response.content}")
```

Comment:
According to your following code, you could use just `if exc.response:` and it would be clear enough.